docs/proposals/secrets-proposal.md

```diff
@@ -27,42 +27,42 @@ To create a new secret, a top level `secrets` key can be used as follows:
 version: "0.1-dev"
 
 services:
-   name: foobar
+ - name: foobar
    containers:
    - image: foo/bar:tag
      ports:
      - port: 8080:80
      env:
-     - foo=bar
-     - travis_creds:
-         secret: ci_secret/travis
+     - name: foo
+       value: bar
+     - name: travis_creds
+       secretRef: ci_secret/travis
      mounts:
      - mountPath: /etc/jenkins.conf
-       secret: ci_secret/jenkins
+       secretRef: ci_secret/jenkins
 
 secrets:
-    ci_secret:
-    - file: /tmp/travis.password
-    - data: jenkins=strongpassword@@
+  - name: ci_secret
+    fromFile:
+    - /tmp/travis.password
+    fromLiteral:
+    - jenkins=strongpassword@@
 ```
 
 The syntax will look like -
 
 ```yaml
 secrets:
-  secret_name:
-  - file: <file path>
-  - data: <data key>=<data value>
-  - data: <data key>=<base64 encoded data>
-    type: base64
-  - data: <data key>=<data value>
-    type: plaintext (default)
-
+  - name: <secret name>
+    fromFile:
+    - <file path>
+    - <file path>
+    fromLiteral:
```

Comment:
I was just thinking that it would be nice to give users option where they don't have to do base64 encoding for every value.
I expect that most user will use some basic string as value, hence two different keys. `value` for plain string and `valueBase64` for advanced use-cases (binary data, multiline etc..) where it's required to have  base64 encoded value.

This was just my rationale behind having `value` and `valueBase64`.
I'm ok with keeping it as `value` that is always base64 encoded.